pkg/descriptions.json

```diff
@@ -155,6 +155,11 @@
     "Description": "Package used to represent the portions of NETCoreApp that come from CoreFx.",
     "CommonTypes": []
   },
+  {
+    "Name": "Microsoft.Private.CoreFx.OOB",
+    "Description": "Package used to represent out-of-band packages that are not included in NetCoreApp.",
```

Comment:
Should we specify that this package is for running tests only?

Reply:
I would not specify it as it could also be used for other cases in our repositories. Ie we already have two different use cases for it, 1) the performance repo and 2) the mono/coreclr/corefx repo.

Reply:
But aren't all of those testing purposes?

Reply:
Right, those two examples usually involve testing but what I'm trying to say is that there could be other use-cases. That's why I'm hesitant adding the word `testing` to the package description.